### ISSUE TITLE
FIx the text effect in Home Page

### ISSUE THREAD

**Issue by TeeWrath** [mindspace-prototype/MindSpace#2]

**Comment by JrG-One** [comment]
Please assign me this issue

**Comment by pankhuri92** [comment]
Hi @TeeWrath, please assign it to me

**Comment by RonitBaranwal** [comment]
Hi @TeeWrath ,I have solved this issue using some JavaScript functionalities . Here's the demo video for this .If its okay for you please reply to my msg then i will make a pr for it .

https://github.com/The-MindSpace/MindSpace-Web/assets/123174522/b43a1c42-3794-4aa0-845b-222746a58fe7

**Comment by TeeWrath** [comment]
@RonitBaranwal Can You Come on Discord, so we can discuss more about this ?

**Comment by RonitBaranwal** [comment]
@TeeWrath Sir, I am trying to contact but ain't getting any response from you.

**Comment by Hacxmr** [comment]
@TeeWrath I am interested in working with this bug. Assign me this.

**Comment by TeeWrath** [comment]
@RonitBaranwal You are assigned for this issue.
Pull already made changes then work. All the best

**Comment by TeeWrath** [comment]
@RonitBaranwal make a PR by tonight, otherwise I will assign this to someone else.

**Comment by TeeWrath** [comment]
@Hacxmr You are assigned this one. Kindly raise a PR by tomorrow.

**Comment by Hacxmr** [comment]
@TeeWrath Sure

**Comment by Hacxmr** [comment]
@TeeWrath I have made a PR. Kindly look at it.

**Comment by Hacxmr** [comment]
Hey! @TeeWrath I am interested in working on UI enhancement.

**Comment by TeeWrath** [comment]
@Hacxmr hey, can u plz give a screen recording of how the changes you made are looking, afterwards we can merge the PR if its correct

**Comment by Hacxmr** [comment]
Yes @TeeWrath Kindly have a look at it and update if any further change is needed

https://github.com/The-MindSpace/MindSpace-Web/assets/129144413/b3bfa8c5-2543-4e01-a1be-ee0ef7b9a0de
